_pages/en_US/seedminer(twn).txt

```diff
@@ -0,0 +1,16 @@
+---
+title: "Seedminer"
```

Comment:
do you want the title to be seedminer or seedminer (twn)

Reply:
made it Seedminer (TWN) k